code/modules/cargo/exports/large_objects.dm

```diff
@@ -171,13 +171,13 @@
 	var/worth = 10
 	var/gases = C.air_contents.gases
 
-	worth += gases[/datum/gas/bz]*4
+	worth += gases[/datum/gas/bz]*2
 	worth += gases[/datum/gas/stimulum]*25
-	worth += gases[/datum/gas/hypernoblium]*1000
-	worth += gases[/datum/gas/miasma]*4
-	worth += gases[/datum/gas/tritium]*7
-	worth += gases[/datum/gas/pluoxium]*6
-	worth += gases[/datum/gas/nitryl]*30
+	worth += gases[/datum/gas/hypernoblium]*50
+	worth += gases[/datum/gas/miasma]*1
+	worth += gases[/datum/gas/tritium]*5
+	worth += gases[/datum/gas/pluoxium]*4
+	worth += gases[/datum/gas/nitryl]*35
```

Comment:
remind me how nitryl's made again and your thought process that warrants bumping up the price a bit when you're throwing everything else out the door?

Reply:
It is needed for other fusions, and tends to be sold less. By increasing its price it its a levy on new methods of fission being made or tried as a bad burn isnt as costly.